crates/sui/tests/ptb_files/simple/move_call.ptb

```diff
@@ -0,0 +1,6 @@
+--gas-budget 1000 
+--assign "a" none --make-move-vec "<0x1::option::Option<u64>>" "[a]" --move-call "0x1::option::is_none" "<u64>" a 
+--assign "a" none --move-call "0x1::option::is_none" "<u64>" none 
+--assign "a" none --move-call "0x1::option::is_none" "<u64>" a 
+--assign "a" @0xf3ecaddd15877e590d66a1181381efda4f49607a29bdfd8b4205cfdfa9b09afc --move-call "0x2::coin::destroy_zero" "<0x2::sui::SUI>" a 
```

Comment:
I don't think this will work? Since I'm guessing that this is an address that is local to your sui CLI, so it won't work for anyone else.

Reply:
Good catch.